src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -126,6 +147,7 @@ pub fn build_ingest(mut cmd: BuiltinCommand) -> Result<IngestAction, String> {
     let format = match cmd.args.string("format")?.as_str() {
         "avro" => Format::Avro {
             schema: cmd.args.string("schema")?,
+            registry_bytes: cmd.args.bool_("registry-bytes").unwrap_or(Ok(true))?,
```

Comment:
Once we settle on a name for the `WITH` option, we should use a name here that is the same or very reminiscent of that name. `registry_bytes` and `records_have_schema_id` are very different names for the same thing.

Reply:
these have both been changed to `confluent_wire_format`